edit_course/operations/clone.py

```diff
@@ -34,8 +36,8 @@ def set_sis(instance: CourseInstance, id: str, enroll: bool) -> None: # pylint:
         return
 
     if coursedata.get('starting_time') and coursedata.get('ending_time'):
-        instance.starting_time = coursedata['starting_time']
-        instance.ending_time = coursedata['ending_time']
+        instance.starting_time = timezone.make_aware(datetime.strptime(coursedata['starting_time'], '%Y-%m-%d'))
+        instance.ending_time = timezone.make_aware(datetime.strptime(coursedata['ending_time'], '%Y-%m-%d'))
```

Comment:
The course starting and ending time can also include time, for example `2023-09-04 10:00`, is this going to break?
Also, why were these changes required?